docs/architecture/platform-versions.md

```diff
@@ -94,13 +94,16 @@ History:
 * We [dropped iOS 11 support] in 2021-04. It was 0.1% of iOS users
   who tried Zulip. We started iOS 12 support at 12.1 because Xcode's
   dropdown for "Deployment Target" didn't have a 12.0.
+* We [dropped Android 5 Lollipop support][dropped-android-l] in
+  2021-08.
 <!-- When updating this, please update `docs/developer-guide.md` as well. -->
 
 [dropped-android-j]: https://chat.zulip.org/#narrow/stream/48-mobile/topic/platform.20versions/near/625585
 [dropped iOS 8 support]: https://chat.zulip.org/#narrow/stream/48-mobile/topic/platform.20versions/near/628412
 [dropped iOS 9 support]: https://chat.zulip.org/#narrow/stream/48-mobile/topic/platform.20versions/near/771761
 [dropped-android-k]: https://chat.zulip.org/#narrow/stream/48-mobile/topic/platform.20versions/near/794551
 [dropped iOS 11 support]: https://chat.zulip.org/#narrow/stream/48-mobile/topic/platform.20versions/near/1165087
+[dropped-android-l]: https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/Updating.20.60minSdkVersion.60/near/1236327
```

Comment:
would be nice if someone could move conversations of that topic, to platform versions, to make this consistent with others.
(I think only admins can move conversations)

Reply:
I don't mind the links not all being to the same topic. In fact it's convenient for the "platform versions" topic to not have much detailed discussion on it, to help make it easy to scan through it for the history of the numbers we've posted there, so I'm happy for the recent discussion to stay on a distinct topic.

It would be good to have a quick message there just recording that we've made this change, linking to the discussion thread and to this PR.